apps/app/src/views/routes/SelectOrg.tsx

```diff
@@ -76,6 +76,7 @@ export const SelectOrg = () => {
               onClick={() => {
                 setOrganization(id);
                 login({
+                  organizationId: id,
```

Comment:
this part is confusing to me, i'm not quite sure why this would be required. My best guess is that as a part of upgrading there was a change in behavior

Reply:
same thing on line 43?

Reply:
also is this something we need in `/Users/paulchristophe/Documents/GitHub/client/packages/elements/src/photon-multirx-form/index.tsx` line 127?